src/OpenFeature.Contrib.Providers.Flagd/Resolver/Rpc/RpcResolver.cs

```diff
@@ -435,27 +441,27 @@ private Service.ServiceClient BuildClientForPlatform(FlagdConfig config)
             if (!useUnixSocket)
             {
 #if NET462_OR_GREATER
-                var handler = new WinHttpHandler();
+                var handler = new System.Net.Http.WinHttpHandler();
```

Comment:
I've fully qualified all these because it's easy to accidentally remove imports used by other platforms, since your IDE's compiler doesn't shows imports your platform isn't using as unused.